ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/port_channel_interfaces.py

```diff
@@ -63,27 +64,31 @@ def port_channel_interfaces(self) -> dict | None:
                                 if port_channel_mode == "active":
                                     parent_interface["lacp_id"] = generate_lacp_id(short_esi)
 
-                        subif_parent_interfaces[interface_name] = parent_interface
+                        subif_parent_interfaces.append(parent_interface)
 
                         for subif in subifs:
                             key = f"{interface_name}.{subif['number']}"
-                            port_channel_interfaces[key] = {
-                                "type": "l2dot1q",
-                                "encapsulation_vlan": {
-                                    "client": {
-                                        "dot1q": {
-                                            "vlan": subif["number"],
+                            port_channel_interfaces.append(
+                                {
+                                    "name": key,
```

Comment:
let's take the opportunity to rename `key` in `subif_name` or something like this for clarity :)

Reply:
updated